src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -59,6 +67,18 @@ public ConfigureOptions(
         [Option("runtime.graphql.depth-limit", Required = false, HelpText = "Max allowed depth of the nested query. Allowed values: (0,2147483647] inclusive. Default is infinity. Use -1 to remove limit.")]
         public int? DepthLimit { get; }
 
+        [Option("runtime.graphql.enabled", Required = false, HelpText = "Configure enable/disable flag for graphQL runtime options.")]
```

Comment:
please mention. "Allowed values: true (default), false." and similarly for others.

Mention restrictions in the uri path.

Reply:
updated